### DISCUSSION TITLE
addLabels not working in GitLab

### DISCUSSION THREAD

**Discussion by matteosilv** [renovatebot/renovate#19385] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34.55.0

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of the platform you run.

15.6.2

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

I'm providing RENOVATE_ADD_LABELS=dependency-update configuration variable, but nothing happens. The label exists in GitLab, and the configuration is clearly shown in debug logs, but renovate does not do anything with it.
I cannot spot any logs about it. Just the printed merged renovate configuration 

### Relevant debug logs

<details><summary>Logs</summary>

```
         "addLabels": ["dependency-update"],
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by secustor** [comment, marked as answer]
Labels are only added to new PRs/Issues currently. https://github.com/renovatebot/renovate/issues/9950

Are they missing on these too?

**Comment by matteosilv** [comment]
So only if the MR is new they will be added. correct?

**Comment by viceice** [comment]
yes
